src/test/java/cz/muni/pa165/teamwhite/formula1/UserTest.java

```diff
@@ -0,0 +1,102 @@
+package cz.muni.pa165.teamwhite.formula1;
+
+import cz.muni.pa165.teamwhite.formula1.dao.UserDao;
+import cz.muni.pa165.teamwhite.formula1.entity.User;
+import cz.muni.pa165.teamwhite.formula1.enums.Role;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.Rollback;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import javax.persistence.*;
+import javax.transaction.Transactional;
+import javax.validation.ConstraintViolationException;
+import java.util.List;
+
+/**
+ * @author Jiří Andrlík
+ */
+@ContextConfiguration(classes = PersistenceSampleApplicationContext.class)
+@TestExecutionListeners(TransactionalTestExecutionListener.class)
+@Transactional
+public class UserTest extends AbstractTestNGSpringContextTests {
+
+    @PersistenceUnit
+    EntityManagerFactory emf;
+
+    @PersistenceContext
+    private EntityManager em;
+
+    @Autowired
+    private UserDao userDao;
+
+
+    @Test
+    public void testCreate(){
+        User kaja = new User("kaja", "CoTeToZajima", Role.MANAGER);
+        userDao.create(kaja);
+        Assert.assertTrue(true);
```

Comment:
:smile:

Reply:
Tests are passing boss! can i go home now?

Reply:
its a test on create... if it doesnt give an exception, I assume it passed :D